README.md

```diff
@@ -43,3 +43,4 @@ Breaking the rules WILL end in your PR being marked as spam. This could cause yo
 I did 34 translations/changes inside Catima glossary and android component for Hindi language.
 - 01-10-2023 [@ziadOUA](https://github.com/ziadOUA) Translated the remaining Android strings in Italian
 - 04-10-2023 https://hosted.weblate.org/translate/catima/catima/hi/?checksum=07f7cfb37b087d85  Translation for Hindi Language in Android.
+- 04-10-2023 [@rudy3333](https://github.com/rudy3333) Added ~100 Polish translations (and still counting!) to Caitma Android Fastlane.
```

Comment:
might want to fix the "Caitma" typo :sweat_smile:

Reply:
Eh, it's fine, this repo only exists to track Hacktoberfest contributions. It's bookkeeping, nothing else :)

Reply:
😭😁sorry lolz it was 1am for me